### PR TITLE
Add support for Helm `--wait-for-jobs` flag

### PR DESCRIPTION
**Pull request by kewats**
Adding support for setting wait-for-jobs in HelmRelease.
 
https://github.com/helm/helm/issues/5948

### CODE REVIEW

**Review by hiddeco — COMMENTED** [review]
Thank you for your pull request :sunflower: 

Think it would be better to enable this by default (as we do with `--wait`), and make the flag an "inverse", so e.g. `DisableWaitForJobs`. This ensures people get the best automated experience by default.

**Comment by kewats** [comment]
Sure, I will make the change. I agree that gives a better experience by default.

**Comment by hiddeco** [comment]
When you are done with the changes, please smash your commits into one as the change is quite small. Thank you :coffee:

File: docs/spec/v2alpha1/helmreleases.md [hunk 1]
```diff
@@ -142,6 +142,11 @@ type Install struct {
 	// +optional
```
Comment (hiddeco) [inline docs/spec/v2alpha1/helmreleases.md:142]: This document is kept for historical reason, but should not include any changes as the `v2alpha1` has never supported these fields. Instead, they should be documented in `v2beta1`.

**Comment by kewats** [comment]
Hi @hiddeco, I have inverted the flag and squashed my commits. Please review, thanks!

File: docs/spec/v2alpha1/helmreleases.md (hunk 1 above)
Comment (kewats) [reply docs/spec/v2alpha1/helmreleases.md]: Done!

**Review by hiddeco — APPROVED** [review]
LGTM, thanks a lot @kewats :1st_place_medal: :100: 

~I think your PR needs a tiny rebase with `main` so that I am able to merge it without any conflicts.~

**Pull request merged**